### ISSUE TITLE
Add methods to ingest strings

### ISSUE THREAD

**Issue by jon-chuang** [scipr-lab/zexe#183]
We should make it easier to input parameters to curves etc., rather than relying on decomposing numbers into hex values etc. It's better for hex values, disastrous for decimals, and overall mistake prone.

**Comment by Pratyush** [comment]
I agree, and maybe this is even possible now that `const fn` s are stable.
